packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -374,6 +379,25 @@ export function getCurrentEventPriority(): * {
   return getEventPriority(currentEvent.type);
 }
 
+export function logRecoverableError(
+  config: ErrorLoggingConfig,
+  error: mixed,
+): void {
+  const onRecoverableError = config;
+  if (onRecoverableError !== null) {
+    // Schedule a callback to invoke the user-provided logging function.
+    scheduleCallback(IdlePriority, () => {
+      onRecoverableError(error);
+    });
+  } else {
+    // Default behavior is to rethrow the error in a separate task. This will
+    // trigger a browser error event.
+    scheduleCallback(IdlePriority, () => {
+      throw error;
```

Comment:
There's a handleErrorInNextTick function in this file that has different semantics but I guess is similarly a fallback default behavior? Should they be unified?

Reply:
Looks like that method is only used by the `queueMicrotask` fallback... I'm unsure why that's necessary, to be honest. Feels wrong.